cfme/tests/test_login.py

```diff
@@ -49,8 +49,6 @@ def test_login(context, method, appliance):
 @pytest.mark.tier(2)
 @pytest.mark.sauce
 @pytest.mark.parametrize('context', [ViaUI])
-# BZ 1632718 is only relevant for Chrome browser
-@pytest.mark.meta(blockers=[BZ(1632718)])
```

Comment:
Can you add a `automates` marker and also add BZ description to polarion docstring? Even though it's only relevant to chrome browser, I think it's better to put it there.

Reply:
We are getting the correct error message but not as per the BZ so I removed that BZ blocker and updated error message.

Reply:
I'm not saying anything about the fix. It's just that this test was blocked by this BZ at some point in time which is why I think it is a good idea to keep a note of the BZ.

Reply:
Done